project/plugins.sbt

```diff
@@ -1,9 +1,5 @@
-addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.3")
+addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.15.0")
 
-addSbtPlugin("io.spray" % "sbt-revolver" % "0.7.2")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
 
-addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.2.0")
-
-addSbtPlugin("com.github.tkawachi" % "sbt-repeat" % "0.1.0")
-
-addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.2")
\ No newline at end of file
+addSbtPlugin("com.github.sbt" % "sbt-jacoco" % "3.3.0")
```

Comment:
* `sbt-assembly` and `sbt-revolver` upgrades are necessary for sbt 1.4 compatibility.
* `sbt-repeat` and `sbt-dependency-graph` were only here to assist with occasional debugging/developer goodness tasks. Let's not include them by default; individual devs can modify plugins as needed during debugging sessions.
* `sbt-jacoco`: this upgrade is not strictly necessary as part of this sbt upgrade but felt worth it.